homeassistant/components/homekit/type_sensors.py

```diff
@@ -72,6 +71,7 @@ class SI(NamedTuple):
     format: Callable[[bool], int | bool]
 
 
+DEVICE_CLASS_CO2: Final = "carbon_dioxide"
```

Comment:
I have created a local constant, but maybe `BINARY_SENSOR_SERVICE_MAP` shouldn't be using home assistant constants anyway?